source/content/guides/decoupled-sites/02-considerations.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Pantheon Decoupled
+subtitle: Considerations
+description: What you need to know to get started with Pantheon Decoupled.
+categories: [platform]
+tags: [webops, workflow, decoupled]
+contributors: [joa-pan]
+type: guide
+layout: guide
+showtoc: true
+anchorid: considerations
+permalink: docs/guides/decoupled-sites/considerations/
+editpath: decoupled-sites/02-considerations.md
+reviewed: "2022-07-31"
+---
+
+You need to ensure you system has the correct components to use Pantheon Decoupled. 
+
+## Components for Pantheon Decoupled
+
+The following components are needed to run Pantheon Decoupled: 
+
+* NodeJS: Installing NVM using Homebrew is recommeded for Mac users.
+
+* Lando: Install the latest release of Lando. Lando ships with a recommended version of Docker Desktop if you do not already have it installed.
+
+* The followig are recommended when using Pantheon Decoupled:
+
+  * VS Code - Other IDEs can be used, but our projects ship with suggested plugins and example settings for VSCode.
+
+* Optionally, the following tools are included in the Lando VM, but can be useful to have installed for use without Lando:
+
+  * PHP - Install using Homebrew on Mac to not conflict with the PHP version that comes with your operating system.
+
+  * Composer - Install composer globally.
+
+  * Terminus - Terminus is needed to update build tools for Pantheon Decoupled.
```

Comment:
Backend - what is required: 
- Terminus
- PHP 
- Lando (develop against deployed Pantheon site)

Frontend -
- Node JS

Backend what is optional :
- Composer (build tools flow)
- CMS backend ( data from another API that is not Drupal or Wordpress) - Import Repo w/o starter kit